### ISSUE TITLE
IP address converter (and probably for other non-core ES field types)

### ISSUE THREAD

**Issue by maniel** [Graylog2/graylog2-server#1164]
Hi
Since ES have dedicated field type for IP addresses, can we have a converter for it?
with it we could use more sophisticated queries like `src_ip:[192.168.0.3 TO 192.168.0.30]`which is not possible when it is stored as a string.

**Comment by henrikjohansen** [comment]
+1

**Comment by zolty87** [comment]
+1

**Comment by jalogisch** [comment]
You need to create a custom mapping for the field where you store the the IP. After that you requested search is possible.

http://docs.graylog.org/en/2.4/pages/configuration/elasticsearch.html#custom-index-mappings

**Comment by maniel** [comment]
yeah, that's what I've done initially, but still, it's a workaround, not graylog's builtin mechanism (hence this feature request), a converter (just mapping a field type, that's all) would be more far more user friendly, or at least a method to easily edit custom mappings would be nice, not everyone is curl/json master and even with tools like postman its a bit of trial and error

**Comment by joschi** [comment]
Refs #1443
Refs #4425
